lib/pages/portal/service/website_provider.dart

```diff
@@ -273,7 +297,14 @@ class WebsiteProvider with ChangeNotifier {
       if (uid != null) {
         final DocumentReference ref =
             FirebaseFirestore.instance.collection('users').doc(uid);
-        final QuerySnapshot qSnapshot = await ref.collection('websites').get();
+
+        final QuerySnapshot qSnapshot = sources.isNotEmpty
+            ? await ref
+                .collection('websites')
+                .where('source', whereIn: sources)
+                .where('source', isEqualTo: 'official')
```

Comment:
Looks smelly to me as well, it would the equivalent of
`SELECT * FROM ... WHERE source IN [...] and source === 'official' `  (if I understand correctly)

@AdrianMargineanu  please check this

Reply:
It looks like a copy past mistake